### PR TITLE
adding histogrammar python package

### PR DESCRIPTION
**Pull request by davidlange6**
http://histogrammar.org 

I didn't manage to test within an actual CMSSW release - so likely there will be revisions to the PR after this one gets integrated.

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: cmssw-tool-conf.spec [hunk 1]
```diff
@@ -175,6 +175,7 @@ Requires: freetype-toolfile
 Requires: tkonlinesw-toolfile
 Requires: py2-cx-oracle-toolfile
 Requires: oracle-toolfile
+Requires: histogrammar
```
Comment (smuzaffar) [inline cmssw-tool-conf.spec:178]: you need to add dependency on histogrammar-toolfile

**Comment by rovere** [comment]
My I naively ask who's the customer of this new external?

**Comment by smuzaffar** [comment]
@davidlange6 , IB/CMSSW_8_1_X/stable is also closed one :-) you need to use IB/CMSSW_8_1_X/gcc530.
I will update cms-bot to close PR for cmsdist branches which are not in use any more.

**Comment by davidlange6** [comment]
@smuzaffar - I was actually more fooled by the forward porting matrix where stable appears to be at the bottom of the stack. Maybe we should also remove that (Or maybe it was commented out and I just didn't notice)..

anyway, will try again today...

**Comment by smuzaffar** [comment]
you are right, we should have remove stable to gcc530 forward port map (done now).

**Comment by smuzaffar** [comment]
closing it was it is not an active branch.

**Pull request closed without merging**